src/library/treeitemmodel.cpp

```diff
@@ -193,3 +193,9 @@ TreeItem* TreeItemModel::getItem(const QModelIndex &index) const {
     }
     return m_pRootItem;
 }
+
+void TreeItemModel::triggerRepaint() {
+    //todo: find a nicer way to do this. Works as it should but kind of hackish
```

Comment:
Yes I did (just now once again to be sure). It has no effect. The Widget will not repaint.

I have a solution now without the sideeffect of moving selection by doing
`beginInsertRows(QModelIndex(), rowCount(), rowCount());`
`endInsertRows()`

but that also isn't the nice way...